client/gulpfile.js

```diff
@@ -22,7 +22,7 @@ gulp.task('copy_language_server', function () {
 })
 
 gulp.task('build_extension', function (callback) {
-  exec('node ./node_modules/vsce/out/vsce package',
+  exec('node ./node_modules/vsce/out/vsce package --baseContentUrl https://github.com/jpogran/puppet-vscode/client --baseImagesUrl https://github.com/jpogran/puppet-vscode/client',
```

Comment:
Following instructions [here](https://code.visualstudio.com/docs/extensions/publish-extension#_marketplace-integration) this overrides the default URL path generated by vsce, otherwise it does not account for our client subfolder